documentation/at/vo_macos.md

```diff
@@ -6,6 +6,13 @@ VoiceOver is a popular screen reader that is free and built in to MacOS.
 
 VoiceOver is built in to MacOS and can be enabled from the System Preferences app.
 
+## How to find your version
+
+1. Press `command+space bar` to open search
```

Comment:
Might be easier to just click the apple logo on the top-left corner and select "About This Mac"

Reply:
Good point, I pushed a commit to address this. I was trying to avoid depending on [sensory characteristics](https://www.w3.org/WAI/WCAG21/Understanding/sensory-characteristics.html).